src/test/java/pkgGame/SudokuTest.java

```diff
@@ -81,15 +81,15 @@ public void isValidValue_Test2() throws Exception {
 		Sudoku s;
 		
 		try {
-			Class<?> c = Class.forName("pkgGame.Sudoku");
+			Class<?> c = Class.forName("pkgGame.Sudoku.Cell");
 			Constructor constructor = c.getConstructor(new Class[] { int[][].class });
 			constructor.setAccessible(true);
 			s = (Sudoku) constructor.newInstance(puzzle);
 			
 			Method mSetCells = c.getDeclaredMethod("setCells", new Class[] { int.class });
 			Method mIsValidValue = c.getDeclaredMethod("isValidValue", new Class[] { int.class });
 			
-			mSetCells.invoke(s, null);
+			mSetCells.invoke(c, null);

```

Comment:
This should remain `s` instead of changing it to the variable `c`. You can't invoke from the context of a constructor.